demos/sample_azure_iot_adu/sample_azure_iot_adu.c

```diff
@@ -122,6 +122,12 @@
  */
 #define sampleazureiotSUBSCRIBE_TIMEOUT                       ( 10 * 1000U )
 
+/**
+ * @brief Timeout for downloading the update image and calling the
+ * AzureIoTHubClient_ProcessLoop() function
```

Comment:
Is it always assumed that these timeouts are in seconds, or could we add that to the description?